### ISSUE TITLE
Use Lighthouse CI without repo

### ISSUE THREAD

**Issue by ggarnier** [GoogleChrome/lighthouse-ci#921]
I'd like to run periodic checks - say 3 times a day - on some URLs, and then post them to Lighthouse CI, so I can check the performance history independent of code changes. But looks like it doesn't support this kind of usage. Each run is necessarily attached to a commit, and I can't upload 2 reports on the same commit.

Is it possible to do something like this?

**Comment by ggarnier** [comment]
Reading the docs, I've just found this info about [build context](https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/configuration.md#build-context):

> you're running without a git repo, in a Jenkins environment, a CI provider we haven't documented yet, or are just running into errors, you can control the build context yourself.

Following the docs, I made it work setting these env vars when calling `lhci upload`:

```sh
LHCI_BUILD_CONTEXT__CURRENT_HASH="1" \
LHCI_BUILD_CONTEXT__COMMIT_MESSAGE="test 1" \
LHCI_BUILD_CONTEXT__AUTHOR="myself" \
LHCI_BUILD_CONTEXT__AVATAR_URL="<some valid image url>" \
LHCI_BUILD_CONTEXT__COMMIT_TIME="2023-06-30T11:00:00-03:00" \
lhci upload
```

**Comment by Tdmain** [comment]
Hello
The app has google in there I ask for us to apply another service I don’t want google on our page. I need everyone to make sure no other company’s mainstream company’s get to our website, they’re intergrading and that’s not good for the site. I’d like google to be taken out of the deal.

Sent from my iPhone

On Jun 30, 2023, at 12:21 PM, Guilherme Garnier ***@***.***> wrote:

﻿

Closed #921<https://github.com/GoogleChrome/lighthouse-ci/issues/921> as completed.
